addons/l10n_it_edi_sdicoop/models/res_config_settings.py

```diff
@@ -7,6 +7,21 @@ class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     is_edi_proxy_active = fields.Boolean(compute='_compute_is_edi_proxy_active')
+    l10n_it_edi_sdicoop_demo_mode = fields.Selection(
+        [('demo', 'Demo'), ('prod', 'Official')],
+        compute='_compute_l10n_it_edi_sdicoop_demo_mode',
+        inverse='_set_l10n_it_edi_sdicoop_demo_mode',
+        readonly=False)
+
+    @api.depends("is_edi_proxy_active")
+    def _compute_l10n_it_edi_sdicoop_demo_mode(self):
```

Comment:
weird depends.  It might work by removing it.

Reply:
It's a `compute` on the settings... it won't get updated if you don't use `depends`, tested with @oco-odoo